src/Http/Traits/RequestTrait.php

```diff
@@ -37,7 +37,30 @@ public function getPaginator(): Paginator
      */
     public function getIncludes(): array
     {
-        return explode(',', $this->query('include', ''));
+        $includes = [];
+        foreach (explode(',', $this->query('include', '')) as $include) {
+            $this->addInclude($include, $includes);
+        }
+
+        return $includes;
+    }
+
+    /**
+     * @param string $include
+     * @param array  $includes
+     */
+    private function addInclude(string $include, array &$includes): void
+    {
+        if (strpos($include, '.') === false) {
+            $includes[] = $include;
+            return;
+        }
+
+        $parentInclude = strtok($include, '.');
+        if (!isset($includes[$parentInclude])) {
+            $includes[$parentInclude] = [];
+        }
+        $this->addInclude(strtok(""), $includes[$parentInclude]);
```

Comment:
`strtok("")` when I dump it I see `false`, can we just use `false` here?

Reply:
If you just dump `strtok("")` withouth having called it first with a second argument, you will get `false` indeed. In this case however it will return the remaining string after the `'.'`.
```
strtok('aaa.bb.cccc', '.'); // Returns 'aaa'
strtok(''); // Returns 'bb.ccc'
```
It's one of those weird PHP functions.

Also, if it were to return `false`, an exception would be thrown because only strings are allowed in the method call.